packages/scss/src/components/_button.scss

```diff
@@ -168,10 +180,24 @@
 
 	&.mod-link {
 		background-color: transparent;
-		color: _color("text.default");
 
-		// default color
-		@include buttonLinkColoring("button.default-palette");
+		// default colors
+		color: _color("grey", 700);
+		
+		&:hover {
+			box-shadow: none;
+			background-color: _color("grey", "50");
+		}
+
+		&:focus {
+			background-color: _color("grey", "50");
+			box-shadow: 0 0 0 4px _color("primary", "100"), inset 0 0 0 1px _color("primary", "300");
+		}
+
+		&:active {
+			background-color: _color("grey", "100");
+			box-shadow: 0 0 0 4px _color("primary", "100"), inset 0 0 0 1px _color("primary", "300");
+		}
```

Comment:
With the default palette the `box-shadow` has to use a different color so I cannot use the `buttonLinkColoring` mixin here.